.gitignore

```diff
@@ -45,3 +45,6 @@ bin/
 
 # vscode
 .vscode
+
+# idea
+.idea
```

Comment:
Nit, should have a new line as the final character of each file

Reply:
Got it, I have Fixed it!